UnitTests/BraintreePayPalTests/BTPayPalDriver_Tests.swift

```diff
@@ -1,9 +1,8 @@
 import XCTest
 import BraintreePayPal
 import BraintreeTestShared
-import SafariServices
 
-class BTPayPalDriver_Checkout_Tests: XCTestCase {
+class BTPayPalDriver_Tests: XCTestCase {
```

Comment:
We consolidated `BTPayPalDriver_BillingAgreement_Tests` and `BTPayPalDriver_Checkout_Tests` into one file to avoid test duplication.